src/Tizen.NUI.Components/Controls/CheckBoxGroup.cs

```diff
@@ -31,15 +31,13 @@ namespace Tizen.NUI.Components
     /// </code>
     /// <since_tizen> 6 </since_tizen>
     /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-    [EditorBrowsable(EditorBrowsableState.Never)]
     public class CheckBoxGroup : SelectGroup
```

Comment:
Do we need to make this Public now?  Wondering if a SelectGroup is really needed, instead Buttons could be grouped by the parent container they are added too.   Currently grouped buttons need adding to a parent View and also added to a SelectGroup. I'm assuming that grouping buttons would need a visual (a box) indication not just conceptual group.